axie-scholar-utilities/source/axie/utils.py

```diff
@@ -13,6 +13,7 @@
 
 USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1944.0 Safari/537.36" # noqa
 TIMEOUT_MINS = 5
+AXIE_CLAIM_COOLDOWN_SECONDS = 60 * 60 * 24 * 14 # Timezone agnostic comparisons
```

Comment:
60s * 60min * 24h * 14days right? maybe add somehting like (14 days in seconds) ? So it's even easier to understand?